src/plugins/recurring-issues/index.js

```diff
@@ -21,6 +21,9 @@ The next scheduled release will occur on ${releaseDate.format("dddd, MMMM Do, YY
 
 - [ ] Remove the 'tsc agenda' label on this issue
 - [ ] Review open pull requests and merge any that are [ready](https://eslint.org/docs/maintainer-guide/pullrequests#when-to-merge-a-pull-request)
+- [ ] Release ``@eslint/js`` with a minor version update in [Jenkins](https://jenkins.eslint.org)
+- [ ] Verify if there are other packages (i.e., ``espree``, ``eslint-scope``, ``eslint-visitor-keys``) that need to be released first
+- [ ] Update ``package.json`` in the ``eslint`` repo with new versions from the preceding step (create and merge a pull request)
```

Comment:
Also:

- `@eslint/js` doesn't necessarily have to be a minor version update. It should match the eslint version, so it could also be patch or major, depending on what version of `eslint` is going to be released. (now, this raises a question of what to do with `@eslint/js` on `eslint` preleases)
- The third step says "update package.json with new versions from the preceding _step_". Maybe _steps_, to include updating package.json with the new version of `@eslint/js`?

Reply:
Well that's what I get for rushing to finish before dinner. The syntax highlighting lied!

Updated the checklist descriptions per your feedback.